### PR TITLE
improved subfolder-documentation

### PR DESCRIPTION
**Pull request by Iridias**
After spending half a day in frustration before finally getting rocket.chat running in a subfolder, I felt that I should share my findings with others.
Thus I improved the - to that point not very helpful - documentation about subfolder-support.

### CODE REVIEW

File: 3. Installation/4. Manual Installation/Running in a sub folder.md [hunk 1]
```diff
@@ -1,6 +1,40 @@
...
+If you compare it to the lines provided in the above linked reverse proxy documentation, you may notice a difference in the RewriteRule - there is an additional `/var/www`.
+That must be changed to your DocumentRoot (if it's not already your DocumentRoot).
+If you pass the whole path, including the DocumentRoot to the rocket.chat server, you'll get a 404 response saying
+```
+Unknown path
+```
+because you configured `/chat` as subfolder (actually it's parsed out of the ROOT_URL you provided) but the rocket.chat server gets a request for `/var/www/chat`, detects a missmatch and returns the above mentioned error.
+
+**Additional notes:**
+- Using the RewriteBase Directive won't work - I tried it
```
Comment (geekgonecrazy) [inline 3. Installation/4. Manual Installation/Running in a sub folder.md]: Maybe remove the "I tried it" bit here.

File: 3. Installation/4. Manual Installation/Running in a sub folder.md [hunk 2]
```diff
@@ -10,5 +44,4 @@ For example:
 ROOT_URL=https://your.domain.com/chat/ node main.js
 ```
 
-## Configuring the application
-Set the `Site URL` under `Admin > General` to your **domain only** like `https://your.domain.com/`
+If you followed the manual installation guide, you may already have configured (and persisted) this evironment variable - in this case you can ignore this advice.
```
Comment (geekgonecrazy) [inline 3. Installation/4. Manual Installation/Running in a sub folder.md]: `evironment variable` -> `environment variable`

**Review by geekgonecrazy — CHANGES_REQUESTED** [review]

**Comment by geekgonecrazy** [comment]
@Iridias thanks for contributing these improvements.  Hopefully this will help others.  Left a couple of small notes

File: 3. Installation/4. Manual Installation/Running in a sub folder.md [hunk 3]
```diff
@@ -1,14 +1,47 @@
...
+To run in a sub folder you should pass the environment variable `ROOT_URL` to the process.
 For example:
 
 ```shell
 ROOT_URL=https://your.domain.com/chat/ node main.js
 ```
 
-## Configuring the application
-Set the `Site URL` under `Admin > General` to your **domain only** like `https://your.domain.com/`
+If you followed the manual installation guide, you may already have configured (and persisted) this evironment variable - in this case you can ignore this advice.
```
Comment (geekgonecrazy) [inline 3. Installation/4. Manual Installation/Running in a sub folder.md]: You got the previous instance of this but can you change:

`evironment variable` -> `environment variable`

**Review by geekgonecrazy — CHANGES_REQUESTED** [review]

File: 3. Installation/4. Manual Installation/Running in a sub folder.md (hunk 3 above)
Comment (Iridias) [reply 3. Installation/4. Manual Installation/Running in a sub folder.md]: Oops, my bad. Commit incoming.

**Comment by rafaelks** [comment]
@geekgonecrazy Can you review it again, please?

**Review by geekgonecrazy — APPROVED** [review]

**Comment by geekgonecrazy** [comment]
@Iridias thanks!  Last thing is to click through and review the CLA and we'll be good to merge :+1:

**Review by docsncode — APPROVED** [review]

**Pull request merged**